ks3-standard/src/commonMain/kotlin/io/ks3/standard/base64/Encoding.kt

```diff
@@ -0,0 +1,148 @@
+/*
+ *  Licensed to the Apache Software Foundation (ASF) under one or more
+ *  contributor license agreements.  See the NOTICE file distributed with
+ *  this work for additional information regarding copyright ownership.
+ *  The ASF licenses this file to You under the Apache License, Version 2.0
+ *  (the "License"); you may not use this file except in compliance with
+ *  the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package io.ks3.standard.base64
+
+/**
+ * @author Alexander Y. Kleymenov
+ * https://github.com/square/okio/blob/master/okio/src/commonMain/kotlin/okio/-Base64.kt
+ */
+
+internal const val BASE64_CHARS = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/"
+internal val BASE64 = BASE64_CHARS.toByteArray(Charsets.UTF_8)
```

Comment:
toByteArray is JVM only... I had a quick look at seeing if we could copy in the Okio implementation, but that's multiplatform and it's spread out a little bit. Maybe it's possible, or we could try refactoring the code here. Or just add a dependency on Okio